### PR TITLE
fix(CI): add push when buildx for ACR

### PR DESCRIPTION
**Pull request by sebdevaux**
## :man: :woman:@mentions of the person or team responsible for reviewing proposed changes

This is a nice example: https://github.com/gravitee-io/gravitee-access-management/pull/1822

### CODE REVIEW

File: .circleci/workflows.yml [hunk 1]
```diff
@@ -693,14 +683,9 @@ jobs:
             export IMAGE_TAG_LATEST=graviteeio.azurecr.io/am-management-ui:${TAG_LATEST}
             echo "export IMAGE_TAG=$IMAGE_TAG" >> $BASH_ENV
             echo "export IMAGE_TAG_LATEST=$IMAGE_TAG_LATEST" >> $BASH_ENV
```
Comment (leleueri) [inline .circleci/workflows.yml:685]: Not useful anymore if both commands are merged

File: .circleci/workflows.yml [hunk 2]
```diff
@@ -674,14 +669,9 @@ jobs:
             export IMAGE_TAG_LATEST=graviteeio.azurecr.io/am-gateway:${TAG_LATEST}
             echo "export IMAGE_TAG=$IMAGE_TAG" >> $BASH_ENV
             echo "export IMAGE_TAG_LATEST=$IMAGE_TAG_LATEST" >> $BASH_ENV
```
Comment (leleueri) [inline .circleci/workflows.yml:671]: Not useful anymore if both commands are merged

File: .circleci/workflows.yml [hunk 3]
```diff
@@ -657,14 +657,9 @@ jobs:
             export IMAGE_TAG_LATEST=graviteeio.azurecr.io/am-management-api:${TAG_LATEST}
             echo "export IMAGE_TAG=$IMAGE_TAG" >> $BASH_ENV
             echo "export IMAGE_TAG_LATEST=$IMAGE_TAG_LATEST" >> $BASH_ENV
```
Comment (leleueri) [inline .circleci/workflows.yml:659]: Not useful anymore if both commands are merged

**Review by leleueri — CHANGES_REQUESTED** [review]

File: .circleci/workflows.yml (hunk 1 above)
Comment (sebdevaux) [reply .circleci/workflows.yml]: removed

File: .circleci/workflows.yml (hunk 2 above)
Comment (sebdevaux) [reply .circleci/workflows.yml]: removed

File: .circleci/workflows.yml (hunk 3 above)
Comment (sebdevaux) [reply .circleci/workflows.yml]: removed

**Review by leleueri — APPROVED** [review]

**Pull request merged**